bigtable-hbase-1.x-parent/bigtable-hbase-tools/README.md

```diff
@@ -0,0 +1,104 @@
+#HBase Tools
+
+This folder contains tools to help HBase users with migrations to Cloud Bigtable.
+Pipelines to import and export data are under [bigtable-beam-import](bigtable-dataflow-parent/bigtable-beam-import/README.md).
+
+## Setup 
+
+To use the tools in this folder, you can download them from the maven repository, or
+you can build them using Maven.
+
+[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
+### Download the jars
+
+todo: get correct link, also should I provide a wget command?
+
+Download [the Bigtable tools jars](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.14.1/bigtable-beam-import-1.14.1-shaded.jar), which is an aggregation of all required jars.
+
+### Build the jars
+Go to the top level directory and build the repo then return to this sub directory.
+
+```
+cd ../../
+mvn clean install -DskipTests=true
+cd bigtable-hbase-1.x-parent/bigtable-hbase-tools
+```
+
+## Schema Translation tool 
+This utility tool will create tables in Cloud Bigtable based on the tables in an HBase cluster.
+You specifiy a name regex and it will copy column families, garbage collection rules,
+and table splits.
+ 
+todo: confirm what it copies?   
+
+todo: what is the regex format?
```

Comment:
What do you mean? it is the same has hbase listTable regex. https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/Admin.html#listTableDescriptors-java.util.regex.Pattern-

Reply:
Thank you, added a link to that